demo/java/build.gradle.kts

```diff
@@ -8,11 +8,13 @@ version = "0.0.1"
 
 repositories {
     mavenCentral()
-    maven("https://jitpack.io")
+    // maven("https://jitpack.io")
 }
 
 dependencies {
-    implementation("com.github.ebraminio:astronomy:2bcbf09edd335a42e013135e04c1418d0bdcde32")
+    implementation(fileTree("../../source/kotlin/build/libs"))
+    // Or resolve it from jitpack like,
+    //   implementation("com.github.cosinekitty:astronomy:0.0.1")
```

Comment:
Same can be done Android example also.

Reply:
This is good that you put the comment there to show people how to do it from JitPack. I like this approach better than #197 because it is self-contained and serves as part of the unit tests for each commit. In the future, if I rename something and forget to rename it in the Java demo, this will catch it immediately.